RunCodeFormatting.rb

```diff
@@ -70,7 +70,7 @@
       raise e
     end
 
-    if path !~ /\/src\//i && path !~ /\/test\//i ||
+    if path !~ /\/src\//i && path !~ /\/test\//i  && path !~ /thirdparty/i ||
```

Comment:
There is a typo here and missing capitalization, Linux is case sensitive. And this should be right at the top of the loop not here

Reply:
what exactly do you mean with the top of the loop
as in only files that have "ThirdParty" in their path get checked?
or they don't get checked

Reply:
There should be a check like `path =~ /\/ThirdParty/i` after line 22 with a `next` in it to skip all future handling for everything in ThirdParty